### PR TITLE
only test on native

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
I sometimes drop into a debug session and use `pkg-config --exists --debug egl` or similar.

File: recipe/build.sh [hunk 1]
```diff
@@ -10,6 +10,19 @@ pushd sources/shiboken6
 
 if test "$CONDA_BUILD_CROSS_COMPILATION" = "1"
 then
+  if [[ "${build_platform}" == "linux-64" ]]; then
+    # There are probably equivalent CDTs to install if your build platform
+    # is something else. However, it is most common in 2023 to use the x86_64
+    # hardware to cross compile for other architectures.
+    mamba install --yes \
+      --prefix ${BUILD_PREFIX} \
```
Comment (hmaarrfk) [inline recipe/build.sh:18]: I'm surprised this didn't work.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -86,11 +84,33 @@ test:
...
+    - xorg-libxext                      # [(build_platform != target_platform) and linux]
+    - xorg-libxdamage                   # [(build_platform != target_platform) and linux]
+    - xorg-libxrandr                    # [(build_platform != target_platform) and linux]
+    - xorg-libxcomposite                # [(build_platform != target_platform) and linux]
+    - xorg-libxcursor                   # [(build_platform != target_platform) and linux]
+    - xorg-libsm                        # [(build_platform != target_platform) and linux]
+    - xorg-libice                       # [(build_platform != target_platform) and linux]
+    - xorg-libxxf86vm                   # [(build_platform != target_platform) and linux]
+    - xorg-libxtst                      # [(build_platform != target_platform) and linux]
+    - xorg-xf86vidmodeproto             # [(build_platform != target_platform) and linux]
```
Comment (hmaarrfk) [inline recipe/meta.yaml:113]: it is curious to me why any of these need to be added here at all. Aren't they all pulled in automatically?

Comment (jschueller) [reply recipe/meta.yaml:113]: probably not, I'm just desperately trying things out

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

**Pull request merged**